komodor/README.md

```diff
@@ -0,0 +1,68 @@
+# Komodor - Kubernetes Troubleshooting Platform
+
+## Overview
+
+This integration allows Datadog Monitor Alerts to be available in the Komodor Kubernetes troubleshooting platform and to suggest related services based on service connections detected by Datadog.
+
+Komodor tracks changes across your entire K8s stack, analyzes their ripple effect and provides you with the context you need to troubleshoot efficiently and independently.
+
+### Empower on-call teams
+Make the knowledge and expertise that has traditionally been held by only a few, clear and visible to developers, SRE and support teams.
+
+### Understand your kubernetes
+Gain the K8s visibility you are lacking. See your deployments on a timeline with the relevant information: what changed, what code was pushed and by whom.
+
+### Track your system end-to-end
+View data from your Git, config, infra, alerting and other tools, in one centralized and easy-to-understand display, easily link to Datadog metrics with dynamic deployment links that take you directly to the data you need.
+
+### Uncovering the context
+Troubleshoot your microservices based on the most relevant context, connections and dependencies.
+
+### Increase performance and availability
+Detect and resolve issues to prevent escalation. Enable any on-call developer to quickly uncover root causes and get the right teams involved for rapid resolution.
+
+## Setup
+
+Once logged into the Komodor platform you can easily install our pod-based agent on each Kubernetes cluster by using a Helm chart or Kustomize - here is a [quick overview][2] of installing the agent.
+
+Once the agent is install you can begin with the Datadog integrations listed below, when they're completed you can enrichen deployment and service metadata with dynamic links and annotations listed below.
+
+### Datadog integration
```

Comment:
Is a user supposed to set up all three of these integration types, or choose one type that works best? If a user is able to use all of this functionality at once, we should not refer to each bit of functionality as a separate integration.

Reply:
I'll add some clarity to this section, ideally a user sets up all three to have the most complete integration experience so that's the guidance I'll add.